### ISSUE TITLE
[Notifications] Notifications broken with “Cannot read property 'imageShown' of undefined”

### ISSUE THREAD

**Issue by mike-cr** [urbit/urbit#3921]
**Describe the bug**
Notifications broken with “Cannot read property 'imageShown' of undefined” after a notification containing an image I think. 

**To Reproduce**
1. Maybe post an image in a channel you've unmuted???
2. Click on Notifications
3. See error and stack trace

**Expected behavior**
Should not crash

**Screenshots**
`“Cannot read property 'imageShown' of undefined”`
Stack Trace:
```
TypeError: Cannot read property 'imageShown' of undefined
    at r.value (https://landscape.tinnus-napbus.xyz/~landscape/js/bundle/index.a3923bdec6badc072f30.js:184:56509)
    at Pa (https://landscape.tinnus-napbus.xyz/~landscape/js/bundle/index.a3923bdec6badc072f30.js:37:69875)
    at Na (https://landscape.tinnus-napbus.xyz/~landscape/js/bundle/index.a3923bdec6badc072f30.js:37:69670)
    at gs (https://landscape.tinnus-napbus.xyz/~landscape/js/bundle/index.a3923bdec6badc072f30.js:37:105469)
    at cl (https://landscape.tinnus-napbus.xyz/~landscape/js/bundle/index.a3923bdec6badc072f30.js:37:96716)
    at sl (https://landscape.tinnus-napbus.xyz/~landscape/js/bundle/index.a3923bdec6badc072f30.js:37:96641)
    at Js (https://landscape.tinnus-napbus.xyz/~landscape/js/bundle/index.a3923bdec6badc072f30.js:37:93671)
    at https://landscape.tinnus-napbus.xyz/~landscape/js/bundle/index.a3923bdec6badc072f30.js:37:45313
    at t.unstable_runWithPriority (https://landscape.tinnus-napbus.xyz/~landscape/js/bundle/index.a3923bdec6badc072f30.js:45:3844)
    at qi (https://landscape.tinnus-napbus.xyz/~landscape/js/bundle/index.a3923bdec6badc072f30.js:37:45022)
```
**Desktop (please complete the following information):**
 - OS: Arch linux 64-bit
 - Browser qutebrowser (qtwebengine)
 - Base hash 0v1r.kh31a.3lh29.dkdc0.m61ud.o9v6j.dogmk.i83bn.e063g.k6vjo.lg6c2

**Comment by custom-jonathan** [comment]
Came to submit the same issue. Here's my stack trace:

```
TypeError: Cannot read property 'imageShown' of undefined
    at r.value (https://minder-folden.arvo.network/~landscape/js/bundle/index.a3923bdec6badc072f30.js:184:56509)
    at Pa (https://minder-folden.arvo.network/~landscape/js/bundle/index.a3923bdec6badc072f30.js:37:69875)
    at Na (https://minder-folden.arvo.network/~landscape/js/bundle/index.a3923bdec6badc072f30.js:37:69670)
    at gs (https://minder-folden.arvo.network/~landscape/js/bundle/index.a3923bdec6badc072f30.js:37:105469)
    at cl (https://minder-folden.arvo.network/~landscape/js/bundle/index.a3923bdec6badc072f30.js:37:96716)
    at sl (https://minder-folden.arvo.network/~landscape/js/bundle/index.a3923bdec6badc072f30.js:37:96641)
    at Js (https://minder-folden.arvo.network/~landscape/js/bundle/index.a3923bdec6badc072f30.js:37:93671)
    at https://minder-folden.arvo.network/~landscape/js/bundle/index.a3923bdec6badc072f30.js:37:45313
    at t.unstable_runWithPriority (https://minder-folden.arvo.network/~landscape/js/bundle/index.a3923bdec6badc072f30.js:45:3844)
    at qi (https://minder-folden.arvo.network/~landscape/js/bundle/index.a3923bdec6badc072f30.js:37:45022)
```

**Comment by galenwp** [comment]
Same here. Stack trace looks more or less the same as what others see:

```
TypeError: Cannot read property 'imageShown' of undefined
    at r.value (http://ravmel-ropdyl.arvo.network/~landscape/js/bundle/index.a3923bdec6badc072f30.js:184:56509)
    at Pa (http://ravmel-ropdyl.arvo.network/~landscape/js/bundle/index.a3923bdec6badc072f30.js:37:69875)
    at Na (http://ravmel-ropdyl.arvo.network/~landscape/js/bundle/index.a3923bdec6badc072f30.js:37:69670)
    at gs (http://ravmel-ropdyl.arvo.network/~landscape/js/bundle/index.a3923bdec6badc072f30.js:37:105469)
    at cl (http://ravmel-ropdyl.arvo.network/~landscape/js/bundle/index.a3923bdec6badc072f30.js:37:96716)
    at sl (http://ravmel-ropdyl.arvo.network/~landscape/js/bundle/index.a3923bdec6badc072f30.js:37:96641)
    at Js (http://ravmel-ropdyl.arvo.network/~landscape/js/bundle/index.a3923bdec6badc072f30.js:37:93671)
    at http://ravmel-ropdyl.arvo.network/~landscape/js/bundle/index.a3923bdec6badc072f30.js:37:45313
    at t.unstable_runWithPriority (http://ravmel-ropdyl.arvo.network/~landscape/js/bundle/index.a3923bdec6badc072f30.js:45:3844)
    at qi (http://ravmel-ropdyl.arvo.network/~landscape/js/bundle/index.a3923bdec6badc072f30.js:37:45022)
```

**Comment by matildepark** [comment]
https://github.com/urbit/urbit/pull/3922 

— 
~haddef-sigwen
https://urbit.org

**Comment by matildepark** [comment]
Fixed in #3922.
